src/components/VErrorSection/VErrorImage.vue

```diff
@@ -70,13 +70,13 @@ export default defineComponent({
 </script>
 
 <style scoped>
-::v-deep .attribution {
+::v-deep(.attribution) {
```

Comment:
The warning in #1634 starts appearing with the update to Vue 2.7, so I just fixed it here.